File: 백준/Bronze/2738. 행렬 덧셈/행렬 덧셈.swift
import Foundation

let NM = readLine()!.split { $0 == " "}.map { Int($0)! }
let (N, M) = (NM[0], NM[1])

var A:[[Int]] = []
var B:[[Int]] = []

for _ in 0 ..< N {
    let input = readLine()!.split { $0 == " "}.map { Int($0)! } // M
    A.append(input)
}

for _ in 0 ..< N {
    let input = readLine()!.split { $0 == " "}.map { Int($0)! } // M
    B.append(input)
}

var result:[[Int]] = []

for i in 0 ..< N {
    var temp: [Int] = []
    for j in 0 ..< M {
        temp.append(A[i][j] + B[i][j])
    }
    result.append(temp)
}

for i in 0 ..< N {
    for j in 0 ..< M {
        print(result[i][j], terminator: " ")
    }
    print()
}

File: 백준/Bronze/2738. 행렬 덧셈/행렬 덧셈.py
import sys
readline = sys.stdin.readline
n, m = map(int, readline().split())

A = [list(map(int, readline().split())) for _ in range(n)]
B = [list(map(int, readline().split())) for _ in range(n)]

for i in range(n):
    for j in range(m):
        print(A[i][j] + B[i][j], end=" ")
    print()
